packages/ez_pgocaml/ez_pgocaml.1.0.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "A simple library to work with pgocaml"
+maintainer: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+authors: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+license: "GPL-3.0-or-later"
+homepage: "https://github.com/ocamlpro/ez_pgocaml"
+bug-reports: "https://github.com/ocamlpro/ez_pgocaml/issues"
+depends: [
+  "dune" {>= "2.0"}
```

Comment:
This should also have the minimum version of OCaml that you expect to support; e.g. `"ocaml" {>="4.06.0"}`

Reply:
Thanks, I should have used `drom` to package it, to avoid all these problems...